installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc

```diff
@@ -23,6 +23,15 @@ include::modules/understanding-agent-install.adoc[leveloffset=+1]
 
 * xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#installation-requirements-platform-none_preparing-to-install-with-agent-based-installer[Requirements for a cluster using the platform "none" option]
 
+* xref:../installing_bare_metal_ipi/ipi-install-prerequisites.adoc#network-requirements-increase-mtu_ipi-install-prerequisites[Increase the network MTU]
+
+* For more information on adding worker nodes to single-node OpenShift clusters, see xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to single-node OpenShift clusters].
+
+[NOTE]
```

Comment:
The note is not indented so maybe adding a plus symbol will collate the note with the last link?